db/schema.rb

```diff
@@ -43,6 +43,8 @@
   create_table "orders", force: :cascade do |t|
     t.string "status"
     t.bigint "user_id"
+    t.datetime "created_at", null: false
```

Comment:
I thought we were leaving the timestamps off of the order table since we don't need them.  Either way is fine, but it might get confusing if we have them there.

Reply:
Card #119 says to include them

```
- the date the order was made
- the date the order was last updated
```

I guess we could somehow go off of order items but that what would be more complicated than this

Reply:
Oh, okay, cool.  Hooray!  Great job!